DEVELOPMENT.adoc

```diff
@@ -0,0 +1,180 @@
+= Development Guide
+:doctype: article
+:toc: macro
+:toclevels: 2
+:toc-title:
+:icons: font
+:table-stripes: uneven
+:imagesdir: assets/images
+:link-repository: https://github.com/d4l-data4life/hc-fhir-sdk-kmp
+// GitHub specific configuration
+ifdef::env-github[]
+:warning-caption: :warning:
+:caution-caption: :fire:
+:important-caption: :exclamation:
+:note-caption: :paperclip:
+:tip-caption: :bulb:
+endif::[]
+
+Guide of our development process, project setup and how to write code.
+
+[discrete]
+==== Table Of Contents
+
+toc::[]
+
+== Prerequisites
+
+* link:https://developer.android.com/studio#downloads[Android Studio 4.2.1]
+* link:https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot[Java 8] | link:https://developer.android.com/studio/write/java8-support[Limitations of Java language features and APIs] | https://jakewharton.com/d8-library-desugaring/[Desugaring]
+* link:https://kotlinlang.org/[Kotlin 1.4.32]
+* link:https://www.python.org/downloads/[Python 3.7.8]
+* Xcode 12.5
+
+== Installation
+
+=== Install Java & Kotlin
+
+You could use link:https://brew.sh/[Homebrew] to install on mac
+
+[source,bash]
+----
+brew tap adoptopenjdk/openjdk
+brew install adoptopenjdk8
+brew install kotlin
+----
+
+OR The easiest way to install Java and Kotlin is to use link:https://sdkman.io/[SDK MAN]
+
+Just open your terminal and enter:
+
+[source,bash]
+----
+curl -s "https://get.sdkman.io" | bash
+----
+
+Follow the setup instructions and verify that your installation works:
+
+[source,bash]
+----
+sdk version
+----
+
+If it shows you something like `5.11.5+713` then your good to continue.
+
+Just install the mentioned dependencies:
+
+[source,bash]
+----
+sdk install java 8.0.292.hs-adpt
+sdk install kotlin 1.4.32
+----
+
+Check your installation by:
+
+[source,bash]
+----
+java -version
+kotlin -version
+----
+
+=== Install Python
+
+It is recommended to use link:https://github.com/pyenv/pyenv[PyEnv] for managing dedicated python versions.
+
+Use link:https://brew.sh/[Homebrew] to install it on mac:
+
+[source,bash]
+----
+brew install pyenv
+----
+
+Make sure you are in the project root and install the correct Python version which is derived from `.python-version` file:
+
+[source,bash]
+----
+pyenv install
+----
+
+=== Android Studio
+
+For better interop with iOS you may install the link:https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile[KMM plugin for Android Studio]]. It let's you run, test and debug shared code on Android and iOS without switching the IDEs.
```

Comment:
`link:https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile[KMM plugin for Android Studio]`